style.css

```diff
@@ -0,0 +1,154 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(235, 228, 228);
+  color: rgb(7, 7, 7);
+}
+
+.container {
+  width: 1300px;
+  margin: 0 auto;
+  position: relative;
+  display: grid;
+  grid-template-columns: 820px 1fr;
+  row-gap: 80px;
+  column-gap: 30px;
+}
+
+header {
+  padding: 40px;
+  grid-column: 1/3;
+  grid-row: 1/2;
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
Try to use CSS class for styling elements of HTML, don't target the element directly.